src/main/java/com/ericsson/eiffelcommons/jenkins/JenkinsManager.java

```diff
@@ -165,18 +155,20 @@ public boolean buildJob(String jobName, String jobToken) throws Exception {
     /**
      * This function that trigger a build on a jenkins job with given body
      *
-     * @param jobName
-     *            :: Name of job as String
-     * @param jobToken
-     *            :: Token used to trigger the job as String
-     * @param body
-     *            :: Body as string "json={param:"param"}"
+     * @param jobName  :: Name of job as String
+     * @param jobToken :: Token used to trigger the job as String
+     * @param body     :: Body as string "json={param:"param"}"
      * @return
      * @throws Exception
      */
-    public boolean buildJobWithFormPostParams(String jobName, String jobToken, String body) throws Exception {
+    @Deprecated
+    public boolean buildJobWithFormPostParams(String jobName, String jobToken, String body)
+            throws Exception {
+        // ! This method currently doesn't work with a fixed GET request as it will fail to set the body !
```

Comment:
How about raising an issue for this?

Reply:
I thought I'd ask @a-breid-ericsson later. We might be able to resolve this.

Reply:
It's been resolved now by sending a POST.